2023-07_Pilot-ER/sections/04-components.adoc

```diff
@@ -14,3 +14,8 @@ Currently, the Standardized Precipitation Index (SPI) is being calculated using
 
 === Wuhan University (WHU)
 Wuhan University (WHU) is a university that plays a significant role in researching and teaching all aspects of surveying and mapping, remote sensing, photogrammetry, and geospatial information sciences in China. In this Climate Resilience Pilot, we will contribute two use-cases: a use-case for drought and wildfire impact, and a use-case for analysis ready data.
+
+=== Ecere Corporation
+
+Ecere is providing a deployment GNOSIS Map Server with a focus on a Sentinel-2 Level 2A data cube. `OGC API - Tiles`, `OGC API - Coverages`, `OGC API - Maps`, `Discrete Global Grid System`, `Common Query Language (CQL2)`, and  `OGC API - Processes - Part 3: Workflows & Chaining` are the supported standards and extensions for this task.
```

Comment:
a deployment **of** GNOSIS Map Server

For the OGC API Standards, use _ _ for italic instead of ` ` pre-formatted.
DGGS should also have _OGC API - _ prefix